docs/_docs/permalinks.md

```diff
@@ -280,38 +294,76 @@ Given a post named: `/2009-04-29-slap-chop.md`
 </table>
 </div>
 
-## Extensionless permalinks
+## Permalink settings for pages and collections
 
-Jekyll supports permalinks that contain neither a trailing slash nor a file
-extension, but this requires additional support from the web server to properly
-serve. When using extensionless permalinks, output files written to disk will
-still have the proper file extension (typically `.html`), so the web server
-must be able to map requests without file extensions to these files.
+The permalink setting in your configuration file specifies the permalink style used for posts, pages, and collections. However, because pages and collections don't have time or categories, these aspects of the permalink style are ignored with pages and collections. 
 
-Both [GitHub Pages](../github-pages/) and the Jekyll's built-in WEBrick server
-handle these requests properly without any additional work.
+For example, a permalink style of `/:categories/:year/:month/:day/:title.html` for posts becomes `/:title.html` for pages and collections.
 
-### Apache
+A permalink style of `pretty` (or `/:categories/:year/:month/:day/:title/`), which omits the file extension and contains a trailing slash, will update page and collection permalinks to also omit the file extension and contain a trailing slash: `/:title/`. But no time or category information will be included.
+
+## Permalinks and default paths
+
+The path to the post or page in the built site differs for posts, pages, and collections:
+
+### Posts
+
+No matter how many subfolders you organize your posts into inside your _posts folder, all posts are pulled out of those subfolders and flattened into the _site's root directory upon build. 
```

Comment:
updated putting _site in backticks, but not posts' since this is not a possessive. it's a folder name.